src/CONST.js

```diff
@@ -492,6 +492,9 @@ const CONST = {
     // 6 numeric digits
     VALIDATE_CODE_REGEX_STRING: /^\d{6}$/,
 
+    // Look for any html/xml tags that would be filtered out by WAF inputrules
```

Comment:
**noting (non-blocking):** nobody is going to know what the "WAF inputrules" are except internal Expensify. Maybe that's fine? Or we could mention something about how the API strips out things that are invalid to contextualize this.

Reply:
I can update this comment, that's a good suggestion.